rust-runtime/aws-smithy-cbor/benches/string.rs

```diff
@@ -0,0 +1,113 @@
+use aws_smithy_cbor::decode::Decoder;
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+
+pub fn str_benchmark(c: &mut Criterion) {
+    // Definite length key `thisIsAKey`.
+    let definite_bytes = [
+        0x6a, 0x74, 0x68, 0x69, 0x73, 0x49, 0x73, 0x41, 0x4b, 0x65, 0x79,
+    ];
+
+    // Indefinite length key `this`, `Is`, `A` and `Key`.
+    let indefinite_bytes = [
+        0x7f, 0x64, 0x74, 0x68, 0x69, 0x73, 0x62, 0x49, 0x73, 0x61, 0x41, 0x63, 0x4b, 0x65, 0x79,
+        0xff,
+    ];
+
+    c.bench_function("definite str_alt", |b| {
```

Comment:
nit: I'd order the benches so that `definite str_alt` comes right after `definite str` etc so they appear next to each other in the report and we can more easily compare.